ape/forms.py

```diff
@@ -1,6 +1,7 @@
 from flask_wtf import FlaskForm
 from wtforms import StringField, SubmitField, EmailField
 from wtforms.validators import DataRequired, Email
+from flask_babel import lazy_gettext as _l
```

Comment:
Do we use it for lazy_gettext as well? This convention (`_l`) was suggested by Miguel Grindberg, as it's a different import. I suggest leave it as is, since `lazy_gettext` is different than `gettext` and to see this one would need to look at import statement.